### ISSUE TITLE
Accessibility Bug: The ‘image’ buttons highlighted are not reachable via keyboard.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3139]
Fundamental core component: "Popover"

Description: The ‘image’ buttons highlighted are not reachable via keyboard.

Expected: All the interactive UI elements should be able to reach via keyboard.

Attached screenshot:

Screenshot 1

![image](https://user-images.githubusercontent.com/32538291/91651346-4e2a7780-eaa9-11ea-8865-1bd938eb9b56.png)

Screenshot 2

![image](https://user-images.githubusercontent.com/32538291/91651350-55518580-eaa9-11ea-9d3d-98bcef033f81.png)

**Comment by puru-hk** [comment]
@Sanjay-1312 We have tested it is working as expected. Please verify from this url (https://deploy-preview-5540--fundamental-ngx.netlify.app/#/core/popover) your side and close this issue.

**Comment by Sanjay-1312** [comment]
Hello Colleagues,

The issue is fixed and Closing the ticket.
The buttons are reachable via keyboard.
URL:https://deploy-preview-5540--fundamental-ngx.netlify.app/#/core/popover

Thanks and Regards
Sanjay
